2024-06-30T09:39:19+0000 WARNI [dtamg_py.utils] ft_restos_pagar_2024 nÃ£o existente em data['fact_tables']
